basxconnect/core/layouts/editperson.py

```diff
@@ -370,20 +378,29 @@ def edit_postal_button(modal):
     )
 
 
+def delete_postal_button(postal):
+    return layout.button.Button(
+        _("Delete"),
+        buttontype="ghost",
+        icon="trash-can",
+        notext=True,
+        hx_post=reverse(
+            DeletePostalAddressView.path(),
```

Comment:
To reduce this pattern a bit I implemented the ```bread.utils.urls.autopath```, ```bread.utils.urls.model_urlname``` and ```bread.utils.urls.resolve_model```functions. They are not always convenient, but in this case they would remove the code for manual naming and an additional method. What do you think?

The idea would be:
You add the url with ```autopath(DeletePostalAddressView, urlname=model_urlname(models.Postal, "ajax_delete"))``` and use ```reverse_model(models.Postal, "ajax_delete")``` to get the URL.

Reply:
The ```"ajax_delete"``` is just a human-readable identifier for the operation on the model, like browse, read, edit, etc.

Reply:
this sounds very reasonable!

Reply:
Ok. Ah, and ModelHref works the same way. It is basically a wrapper for ```model_urlname``` which implement the ```hg.Lazy``` interface, for cases where we e.g. need an id from the current context.

Reply:
done!